app/routes/signup.tsx

```diff
@@ -135,61 +142,198 @@ export const loader: LoaderFunction = async ({request}) => {
   })
 }
 
+const TEAM_MAP = {
+  BLUE: {
+    image: images.alexBlue,
+    label: 'Blue Team',
+    focusClassName: 'ring-team-blue',
+  },
+  RED: {
+    image: images.alexRed,
+    label: 'Red Team',
+    focusClassName: 'ring-team-red',
+  },
+  YELLOW: {
+    image: images.alexYellow,
+    label: 'Yellow Team',
+    focusClassName: 'ring-team-yellow',
+  },
+}
+
+interface TeamOptionProps {
+  team: keyof typeof TEAM_MAP
+  error?: string | null
+  selected: boolean
+}
+
+function TeamOption({team: value, error, selected}: TeamOptionProps) {
+  const team = TEAM_MAP[value]
+
+  return (
+    <div
+      className={clsx(
+        'relative col-span-full mb-3 bg-gray-100 dark:bg-gray-800 rounded-lg focus-within:outline-none ring-team-current ring-offset-4 focus-within:ring-2 lg:col-span-4 lg:mb-0',
+        team.focusClassName,
+        {
+          'ring-2 ring-offset-team-current': selected,
+          'dark:ring-offset-gray-900 ring-offset-white': !selected,
+        },
+      )}
+    >
+      {selected ? (
+        <span className="absolute left-9 top-9 text-team-current">
+          <CheckIcon />
+        </span>
+      ) : null}
+
+      <label className="block pb-12 pt-20 px-12 text-center">
```

Comment:
Let's add a `cursor-pointer` to this. Otherwise people may not realize they can select it.